src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -50,5 +56,11 @@ public EventCard(Event event, int displayedIndex) {
         dateAndTime.setText(event.getStartDate().toString() + " to " + event.getEndDate().toString());
         loc.setText("Location: " + event.getLocation().location);
         description.setText("Description: " + event.getDescription().description);
+        event.getRoles().stream()
+                .sorted(Comparator.comparing(role -> role.roleName))
+                .forEach(role -> roles.getChildren().add(new RoleLabel(role).getRoot()));
+        event.getMaterials().stream()
+                .sorted(Comparator.comparing(material -> material.material))
+                .forEach(material -> materials.getChildren().add(new MaterialLabel(material).getRoot()));
```

Comment:
big brain

Reply:
:D i love my mac

Reply:
someone else did this i wasn't that smart :(